packages/im-ott/src/organisms/filmList/FilmList.js

```diff
@@ -85,13 +101,27 @@ const FilmList = (props) => {
   }
 
   return (
-    <div className={styles.horizontalList}>
+    <div className={styles.mt1}>
```

Comment:
should have a more meaningful name

Reply:
most class names are like these if you check any library, its standard name for mt-1 margin-top: 0.5rem